### PR TITLE
compile and deploy Augur contracts to ganache node running in the tests

### PR DESCRIPTION
**Pull request by robert-e-davidson3**

### CODE REVIEW

File: packages/ethersjs-provider/src/EthersJsonRpcProvider.ts [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { NetworkId } from "@augurproject/artifacts";
```
Comment (robert-e-davidson3) [inline packages/ethersjs-provider/src/EthersJsonRpcProvider.ts:1]: This file won't be committed. It's temporary to make the rest of the code work.

File: packages/ethersjs-provider/src/EthersWeb3Provider.ts [hunk 2]
```diff
@@ -0,0 +1,61 @@
+import { NetworkId } from "@augurproject/artifacts";
```
Comment (robert-e-davidson3) [inline packages/ethersjs-provider/src/EthersWeb3Provider.ts:1]: This file won't be committed. It's temporary to make the rest of the code work.

File: packages/ethersjs-provider/src/index.ts [hunk 3]
```diff
@@ -1 +1,2 @@
-export { EthersProvider } from "./EthersProvider";
```
Comment (robert-e-davidson3) [inline packages/ethersjs-provider/src/index.ts:1]: This file won't be committed. It's temporary to make the rest of the code work.

File: packages/augur-state/src/db/DB.test.ts [hunk 4]
```diff
@@ -1,103 +1,25 @@
 import { TrackedUsers } from "./TrackedUsers";
 import { DB, UserSpecificEvent } from "./DB";
-import { EthersProvider } from "ethers-provider";
+import { EthersWeb3Provider } from "@augurproject/ethers-provider";
```
Comment (robert-e-davidson3) [inline packages/augur-state/src/db/DB.test.ts:3]: This line will probably be different as it's based on temporary code I used while waiting for the proper implementation to be completed. I'm not sure which ticket that has though. @adrake33?

Comment (adrake33) [reply packages/augur-state/src/db/DB.test.ts:3]: Yeah, this line should be `import { EthersProvider, Web3AsyncSendable } from "ethers-provider";`

**Pull request closed without merging**